# If a coin is tossed a sequence of times, what is the probability that the first head will occur after the fifth toss, given that it has not occurred in the first two tosses?

___

# We know that this follows the geometric distribution, and we know that the geometric distribution is memoryless

# This means $P(X>5|X>2) = P(X>3)$

# $P(X>3) = (1-p)^{3} = \frac{1}{8}$

___
### We can support this answer through simulation

In [1]:
import numpy as np

In [8]:
cumsum = 0

for i in range(1,6):
    Px = (0.5)*(0.5**(i-1))
    cumsum += Px
    print('P(X='+str(i)+') = '+str(Px))
    print('P(X>'+str(i)+') = '+str(1-cumsum))
    print('\n')

P(X=1) = 0.5
P(X>1) = 0.5


P(X=2) = 0.25
P(X>2) = 0.25


P(X=3) = 0.125
P(X>3) = 0.125


P(X=4) = 0.0625
P(X>4) = 0.0625


P(X=5) = 0.03125
P(X>5) = 0.03125




## We got the same result for $P(X>3) = 0.125 = \frac{1}{8}$

## We also know $P(X>5|X>3) = \frac{P(X>5 \cap X>2)}{P(X>2)} = \frac{P(X>5)}{P(X>2)} = \frac{0.03125}{0.25} = 0.125 = \frac{1}{8}$